### PR TITLE
[Polygon Edge] Add videos for BLS + Permission smart contract deployment

### PR DESCRIPTION
**Pull request by laviniat1996**

### CODE REVIEW

**Review by Aleksao998 — APPROVED** [review]
LGTM

File: docs/edge/additional-features/permission-contract-deployment.md [hunk 1]
```diff
@@ -19,6 +19,10 @@ Sometimes network operators wish to prevent users from deploying smart contracts
 1. Whitelist addresses for Smart Contract deployment
 2. Remove addresses from the whitelist for Smart Contract deployment
 
+## Video presentation
+
+[![permission contract deployment - video](https://img.youtube.com/vi/yPOkINpf7hg/0.jpg)](https://www.youtube.com/watch?v=yPOkINpf7hg&ab_channel=Trapesys)
```
Comment (Kourin1996) [inline docs/edge/additional-features/permission-contract-deployment.md:24]: query parameter `ab_channel` is not necessary

File: docs/edge/consensus/bls.md [hunk 2]
```diff
@@ -13,6 +13,10 @@ keywords:
 
 BLS is a signature scheme that can aggregate multiple signatures. In Polygon Edge, BLS is used by default in order to provide better security in the IBFT consensus mode. BLS can aggregate signatures into a single byte array and reduce the block header size. Each chain can choose whether to use BLS or not. The ECDSA key is used regardless of whether the BLS mode is enabled or not.
 
+## Video presentation
+
+[![bls - video](https://img.youtube.com/vi/HbUmZpALlqo/0.jpg)](https://www.youtube.com/watch?v=HbUmZpALlqo&ab_channel=Trapesys)
```
Comment (Kourin1996) [inline docs/edge/consensus/bls.md:18]: same as the above

File: docs/edge/additional-features/permission-contract-deployment.md (hunk 1 above)
Comment (laviniat1996) [reply docs/edge/additional-features/permission-contract-deployment.md]: I didn't notice it's there. Fixed 5e781f068ef4d5c433bd955a52f07cab2a2daf65

File: docs/edge/consensus/bls.md (hunk 2 above)
Comment (laviniat1996) [reply docs/edge/consensus/bls.md]: Fixed 5e781f068ef4d5c433bd955a52f07cab2a2daf65

**Pull request merged**